### PR TITLE
add standard field conventions and tooltips

### PR DESCRIPTION
**Pull request by Jbern16**
- Adds Proper tooltip for GRE and GPA fields
- Uses right conventions for standard field name

The conditionals are increasing, we may want to break some of these into there own object.

Possible questions: GPA, GRE (due to the tooltips), zipcode (convention on that is WEIRD).

### CODE REVIEW

File: models/question.rb [hunk 1]
```diff
@@ -59,18 +59,46 @@ def position
...
+    end
+  end
+
   def div_id
     nil
   end
 
+  def title
+    if field_id == "statedGpaRangeField"
+      @title.gsub('GPA', ' <span class="tooltip tooltip--top tooltip--underline" aria-label="Grade Point Average">GPA</span>')
```
Comment (KatyHerrick) [inline models/question.rb]: Is the space before `<span...` intentional? Haven't tested yet but I'd think if you're doing a sub on "GPA", the space originally before GPA will remain there.

Comment (Jbern16) [inline models/question.rb]: we want the space cause of the issue with vu-edu, right? I believe that space will remain when using gsub.

Comment (KatyHerrick) [inline models/question.rb]: Just tested an it creates a duplicate space. It's subtle, but the issue with VU-EDU was 
`<label">What was your undergraduate<span> GPA</span>?</label>` vs
`<label">What was your undergraduate <span>GPA</span>?</label>`

Comment (Jbern16) [inline models/question.rb]: oh! gotcha. makes sense

**Pull request merged**